packages/tx/src/4844/tx.ts

```diff
@@ -172,13 +171,16 @@ export class Blob4844Tx implements TransactionInterface<TransactionType.BlobEIP4
         throw new Error(msg)
       }
     }
-    if (this.blobVersionedHashes.length > LIMIT_BLOBS_PER_TX) {
-      const msg = Legacy.errorMsg(this, `tx can contain at most ${LIMIT_BLOBS_PER_TX} blobs`)
+
+    const limitBlobsPerTx = this.common.param('maxblobGasPerBlock') / this.common.param('blobGasPerBlob')
```

Comment:
This param is only available if we activate EIP 7691 :thinking: Should we introduce a similar value for EIP-4844? :thinking:

Reply:
I think this was intended, have done so.